test/fixtures_s3_test.go

```diff
@@ -0,0 +1,165 @@
+//+build integration fixtures
```

Comment:
please adapt this to a new-style integration test, see `pkg/test/suite_test.go`

Reply:
also here, I followed the referenced files (test files) in the ticket (issue). It'd have been nice mentioning this there. Now I'll have to re-read then re-adapt